README.md

```diff
@@ -52,6 +54,17 @@ If you are new to Graviton and want to understand how to identify target workloa
 |DRAM	|8x DDR4	|8x DDR5	|
 |DDR Encryption	|yes	|yes	|
 
+# x86 monitoring differences
+## Utilities for obtaining processor stats
+|Processor	|x86	|Graviton2,3	|
+|---	|---	|---	|
+|CPU frequency listing	|*lscpu, /proc/cpuinfo, dmidecode*	|*dmidecode*	|
+|*turbostat* support	|Yes	|No	|
+|*i7z* Works	|Yes	|No	|
```

Comment:
works here, support above, but you can bulk all those up together and say no, because of x,y,z.